netbox/extras/forms/model_forms.py

```diff
@@ -50,13 +52,17 @@ class CustomFieldForm(BootstrapMixin, forms.ModelForm):
         required=False,
         help_text=_("Type of the related object (for object/multi-object fields only)")
     )
+    choice_set = DynamicModelChoiceField(
+        queryset=CustomFieldChoiceSet.objects.all(),
+        required=False
+    )
 
     fieldsets = (
         ('Custom Field', (
```

Comment:
Add localization to these fieldset headers

Reply:
These weren't changed as part of the PR